.circleci/config.yml

```diff
@@ -148,12 +152,23 @@ jobs:
       - setup_build_environment
       - snyk/scan:
           additional-arguments: --all-sub-projects --policy-path=.snyk
-  package-charts:
+  release-charts:
     executor: helm
+    working_directory: ~/repo
```

Comment:
working directory not needed, we generally use the default

Reply:
removing this